pom.xml

```diff
@@ -270,6 +271,17 @@
         <artifactId>value</artifactId>
         <version>${version.immutables}</version>
       </dependency>
+      <dependency>
```

Comment:
Why are you declaring the `booster-catalog-service` dependency again?

Reply:
Probably a merge conflict. Fixed